### DISCUSSION TITLE
External links should display an arrow ↗

### DISCUSSION THREAD

**Discussion by Xkonti** [withastro/starlight#1260] [Feature Requests]
### What version of `starlight` are you using?

0.14.0

### What is your idea?

It's common that in wiki/documentation content links to external websites display an arrow next to the link:
[Great website ↗](https://astro.build)

It would be nice to have that feature with configuration to enable/disable it.

### Why is this feature necessary?

I believe it's a feature that is necessary for an integration that focuses on simplifying creation of documentation websites, where it's a common and safety-related feature.

### Participation

- [ ] I am willing to submit a pull request for this feature.

**Comment by delucis** [comment]
Sorry for the slow response on this one. The [“Add icons to external links”](https://docs.astro.build/en/recipes/external-links/) recipe in Astro’s docs should help here!
